### ISSUE TITLE
Upcoming Windows change: 🚀 TLS v1 and v1.1 are going to be disabled by default!

### ISSUE THREAD

**Issue by HotCakeX** [HotCakeX/Harden-Windows-Security#74]
https://techcommunity.microsoft.com/t5/windows-it-pro-blog/tls-1-0-and-tls-1-1-soon-to-be-disabled-in-windows/bc-p/3888775

When this change makes it to the stable builds of Windows, this hardening measure will no longer be necessary and will be removed from the hardening script.

**Comment by Pierre-Gronau-ndaal** [comment]
I do not agree - disable 1.0 and 1.1 everytime for all will or can not patch

**Comment by HotCakeX** [comment]
> I do not agree - disable 1.0 and 1.1 everytime for all will or can not patch

Hello,
All of the tools in this repository are only guaranteed to work on the latest version of the OS, it's in the [requirements](https://github.com/HotCakeX/Harden-Windows-Security#requirements-) too.

**Having an up-to-date OS is more important than using this script.** If your OS version is old and it's vulnerable then no script in the world can fix it. There are many vulnerabilities that are fixed in each monthly update: https://msrc.microsoft.com/update-guide/

This hardening script in the [Windows Update Configurations category](https://github.com/HotCakeX/Harden-Windows-Security#windows-update-configurations) sets the OS to auto update and require a restart 24 hours after the updates are installed.

Here is the official website for Windows 11 22H2 cumulative updates release notes:
https://support.microsoft.com/en-gb/topic/windows-11-version-22h2-update-history-ec4229c3-9c5f-4e75-9d6d-9025ab70fcce

Twitter accounts you can follow and turn on the notification for to know when an update is released and get other security related news

* https://twitter.com/WindowsUpdate
* https://twitter.com/msftsecresponse
* https://twitter.com/MsftSecIntel

**Comment by HotCakeX** [comment]
Closing this for now because Microsoft changed their timeline for disabling TLS 1 and 1.1 in the OS by default, which was set to happen in the build 24H2, and there is currently no date. When that happens, we can review this again and act accordingly.
